src/util/os_utils.js

```diff
@@ -341,12 +340,62 @@ function get_disk_mount_points() {
         });
 }
 
+function _df_parse(stdout) {
+    return stdout
+            .trim()
+            .split(/\r\n|\r|\n/) // split into rows
+            .slice(1) // strip column headers away
+            .map(function(row) {
```

Comment:
maybe use arrow function:
```suggestion
            .map((row) =>  {
```